portal/models/i18n.py

```diff
@@ -0,0 +1,65 @@
+"""Module for i18n methods and functionality"""
+import sys
+import os
+from flask import current_app
+from ..extensions import babel
+from .user import current_user
+from .app_text import AppText
+from .intervention import Intervention
+
+
+def add_to_dict(d,k,o):
+    if k:
+        key = "\"" + k + "\""
+        if key not in d:
+            d[key] = set()
+        d[key].add(o)
+
+
+def get_db_strings():
+    elements = {}
+    query = AppText.query.with_entities(AppText.custom_text, AppText.name)
```

Comment:
sorry to flop around on you - but I'd misread that you only wanted a single column in your query.  I only bother using `with_entities` when it would significantly improve readability or speed.  you might just do `AppText.query()` and then reference the named attribute for more readable code, i.e `entry.custom_text` and `entry.name` in the following line.  Make sense?